src/Models/ShallowWaterModels/calculate_tendencies.jl

```diff
@@ -1,7 +1,6 @@
-import Oceananigans.TimeSteppers: calculate_tendencies!
-
 using Oceananigans.Utils: work_layout
-using ..Models: fields
+using Oceananigans.Models: fields
```

Comment:
It might be best to name this file `calculate_shallow_water_tendencies.jl` just to avoid confusion with another file of the same name in the source.